examples/demo_dlmbl/solution.py

```diff
@@ -905,28 +1191,60 @@ def min_max_scale(input):
     logger=TensorBoardLogger(
         save_dir=log_dir,
         name="phase2fluor",
-        version="wider",
+        version="phase2fluor_low_lr",
         log_graph=True,
     ),
     fast_dev_run=True,
 )  # Set fast_dev_run to False to train the model.
-trainer.fit(phase2fluor_wider_model, datamodule=phase2fluor_data)
+trainer.fit(phase2fluor_model_low_lr, datamodule=phase2fluor_2D_data)
 
 # %% tags=["solution"]
 
 ##########################
 ######## Solution ########
 ##########################
+"""
+You can download the file and place it in the data folder.
+https://public.czbiohub.org/comp.micro/viscy/VSCyto3D/train/raw-and-reconstructed.zarr/
 
-phase2fluor_slow_model = VSUNet(
-    model_config=phase2fluor_config.copy(),
+You can run the following shell script:
+```
+cd ~/data/hek3d/training
+# Download the Zarr dataset recursively (if the server supports it)
+wget -m -np -nH --cut-dirs=4 -R "index.html*" "https://public.czbiohub.org/comp.micro/viscy/VSCyto3D/train/raw-and-reconstructed.zarr/"
+```
+
+"""
+
+data_path = Path()  # TODO: Point to a 3D dataset (HEK, Neuromast)
+BATCH_SIZE = 4
+YX_PATCH_SIZE = (256, 256)
+
+## For 3D training - VSCyto3D
+source_channel = ["Phase3D"]
+target_channel = ["Nucl", "Mem"]
+
+phase2fluor_3D_data = HCSDataModule(
+    data_path,
+    architecture="UNeXt2",
+    source_channel=source_channel,
+    target_channel=target_channel,
+    z_window_size=5,
+    split_ratio=0.8,
     batch_size=BATCH_SIZE,
-    loss_function=torch.nn.functional.l1_loss,
-    # lower learning rate by 5 times
+    num_workers=8,
+    yx_patch_size=YX_PATCH_SIZE,
+    augmentations=augmentations,
+    normalizations=normalizations,
+)
+
+phase2fluor_3D = VSUNet(
+    architecture="UNeXt2",
+    model_config=phase2fluor_config.copy(),
```

Comment:
What does this section do? Also this is now a different model and copying the 2D config wouldn't work.

Reply:
This section was to show the 3D training.